google/bigtable/admin/v2/bigtable_admin_gapic.yaml

```diff
@@ -0,0 +1,300 @@
+  - name: CreateInstance
+    flattening:
+      groups:
+      - parameters:
+        - parent
+        - instance_id
+        - instance
+        - clusters
+    required_fields:
+    - parent
+    - instance_id
+    - instance
+    - clusters
+    request_object_method: true
+    retry_codes_name: non_idempotent
+    retry_params_name: default
+    field_name_patterns:
+      parent: project
+    timeout_millis: 60000
+  - name: GetInstance
+    flattening:
+      groups:
+      - parameters:
+        - name
+    required_fields:
+    - name
+    request_object_method: false
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      name: instance
+    timeout_millis: 60000
+  - name: ListInstances
+    flattening:
+      groups:
+      - parameters:
+        - parent
+    required_fields:
+    - parent
+    request_object_method: true
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      parent: project
+    timeout_millis: 60000
+  - name: UpdateInstance
+    flattening:
+      groups:
+      - parameters:
+        - name
+        - display_name
+    required_fields:
+    - name
+    - display_name
+    request_object_method: true
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      name: instance
+    timeout_millis: 60000
+  - name: DeleteInstance
+    flattening:
+      groups:
+      - parameters:
+        - name
+    required_fields:
+    - name
+    request_object_method: false
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      name: instance
+    timeout_millis: 60000
+  - name: CreateCluster
+    flattening:
+      groups:
+      - parameters:
+        - parent
+        - cluster_id
+        - cluster
+    required_fields:
+    - parent
+    - cluster_id
+    - cluster
+    request_object_method: true
+    retry_codes_name: non_idempotent
+    retry_params_name: default
+    field_name_patterns:
+      parent: instance
+    timeout_millis: 60000
+  - name: GetCluster
+    flattening:
+      groups:
+      - parameters:
+        - name
+    required_fields:
+    - name
+    request_object_method: false
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      name: cluster
+    timeout_millis: 60000
+  - name: ListClusters
+    flattening:
+      groups:
+      - parameters:
+        - parent
+    required_fields:
+    - parent
+    request_object_method: true
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      parent: instance
+    timeout_millis: 60000
+  - name: UpdateCluster
+    flattening:
+      groups:
+      - parameters:
+        - name
+        - serve_nodes
+    required_fields:
+    - name
+    - serve_nodes
+    request_object_method: true
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      name: cluster
```

Comment:
Did you consider doing a resource name for `location`? If we do, we'd probably want to ask the BigTable team how flexible it should be.